src/test/regress/expected/multi_create_table.out

```diff
@@ -62,7 +62,22 @@ CREATE TABLE customer (
 	c_acctbal decimal(15,2) not null,
 	c_mktsegment char(10) not null,
 	c_comment varchar(117) not null);
-SELECT master_create_distributed_table('customer', 'c_custkey', 'append');
+SELECT create_reference_table('customer');
```

Comment:
would you mind adding small comments on why we have both `customer ` and `customer_append` tables? Maybe a single comment at the beginning of the test to explain once?